src/migration-scripts/vpp/3-to-4

```diff
@@ -0,0 +1,34 @@
+# Copyright VyOS maintainers and contributors <maintainers@vyos.io>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this library.  If not, see <http://www.gnu.org/licenses/>.
+
+# Migrate "vpp settings nat44 no-forwarding"
+# to "vpp settings nat44 processing-mode <static+dynamic|static+bypass>"
+
+from vyos.configtree import ConfigTree
+
+base = ['vpp', 'settings', 'nat44']
+base_nat44 = ['vpp', 'nat44']
+
+def migrate(config: ConfigTree) -> None:
+    if not config.exists(['vpp']):
+        # Nothing to do
+        return
+
+    if config.exists(base + ['no-forwarding']):
+        # Delete no-forwarding option from NAT44 settings
+        config.delete(base + ['no-forwarding'])
```

Comment:
Is it better to replace the value explicitly rather than just deleting it?
To `static-dynamic`

This way, people will see how it changed during the upgrade. For new configurations, it will get the default value, which we can change from time to time.